File: Museo Java/OtroObjeto.java
import java.time.LocalDate;

public class OtroObjeto extends ObraDeArte {

    private String descripcion;

    public OtroObjeto(int idObra, String titulo, String autor, String periodo,
                      double valoracion, LocalDate fechaCreacion, LocalDate fechaEntrada,
                      String descripcion, String estado) {
        super(idObra, titulo, autor, periodo, valoracion, fechaCreacion, fechaEntrada, estado);
        this.descripcion = descripcion;
    }

    public OtroObjeto(int idObra, String titulo, String autor, String periodo,
                      double valoracion, LocalDate fechaCreacion, LocalDate fechaEntrada,
                      String descripcion) {
        this(idObra, titulo, autor, periodo, valoracion, fechaCreacion, fechaEntrada, descripcion, "expuesta");
    }

    public String getDescripcion() { return descripcion; }

    @Override
    public String consultarInfo() {
        return String.format("Objeto: '%s' | Autor: %s | Periodo: %s | Descripción: %s | Valoración: $%,.2f | Estado: %s",
                titulo, autor, periodo, descripcion, valoracion, estado);
    }
}

File: MuseoPOO/otro_objeto.py
"""Módulo que define la clase OtroObjeto."""

from datetime import date

from obra_de_arte import ObraDeArte


class OtroObjeto(ObraDeArte):
    """Representa otro tipo de objeto artístico del museo."""

    def __init__(
        self,
        id_obra: int,
        titulo: str,
        autor: str,
        periodo: str,
        valoracion: float,
        fecha_creacion: date,
        fecha_entrada: date,
        descripcion: str,
        estado: str = "expuesta",
    ) -> None:
        super().__init__(
            id_obra, titulo, autor, periodo,
            valoracion, fecha_creacion, fecha_entrada, estado,
        )
        self._descripcion = descripcion

    def get_descripcion(self) -> str:
        return self._descripcion

    def consultar_info(self) -> str:
        return (
            f"Objeto: '{self._titulo}' | Autor: {self._autor} | "
            f"Periodo: {self._periodo} | "
            f"Descripción: {self._descripcion} | "
            f"Valoración: ${self._valoracion:,.2f} | "
            f"Estado: {self._estado}"
        )
